### ISSUE TITLE
Adding more core S3 methods

### ISSUE THREAD

**Issue by strengejacke** [tslumley/svylme#3]
Would it be possible to include some more information in the returned objects from `svylme()`, or adding some more methods? Like `nobs()`, `family()`, maybe the call, the model frame, AIC, ... etc.? Would be nice if some of those functions/information that are easy to add could be implemented.

**Comment by tslumley** [comment]
family() is a bit pointless since it only does linear mixed models. I'll look at some others, though.

 -thomas

Thomas Lumley
Professor of Biostatistics

________________________________________
From: Daniel ***@***.***>
Sent: Thursday, June 27, 2024 7:19 PM
To: tslumley/svylme
Cc: Subscribed
Subject: [tslumley/svylme] Adding more core S3 methods (Issue #3)

Would it be possible to include some more information in the returned objects from svylme(), or adding some more methods? Like nobs(), family(), maybe the call, the model frame, AIC, ... etc.? Would be nice if some of those functions/information that are easy to add could be implemented.

**Comment by strengejacke** [comment]
> family() is a bit pointless since it only does linear mixed models.

I agree for interactive usage. But there might be packages that rely on calling `stats::family()` without knowing the input model.
